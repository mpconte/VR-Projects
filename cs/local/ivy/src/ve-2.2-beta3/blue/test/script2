#
# My second script
#
[stdout] writeln "Hello World!"
[stdout] writeln "Hello to you too!"
