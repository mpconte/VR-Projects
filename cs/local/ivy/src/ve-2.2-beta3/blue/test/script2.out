Hello World!
Hello to you too!
result = ok
